src/DependencyInjection/Providers/GoogleProviderConfigurator.php

```diff
@@ -19,9 +19,11 @@ public function buildConfiguration(NodeBuilder $node)
         // todo - add the comments as help text, and render in README
         $node
             ->scalarNode('access_type')
+                ->defaultValue('')
                 ->info('Optional value for sending access_type parameter. More detail: https://developers.google.com/identity/protocols/OpenIDConnect#authenticationuriparameters')
             ->end()
             ->scalarNode('hosted_domain')
+                ->defaultValue('')
```

Comment:
Hum, I don't know, I used the defaults proposed in the README [here](https://github.com/knpuniversity/oauth2-client-bundle#configuration).
Let me know what is you preference :wink: